### ISSUE TITLE
Duplicate images

### ISSUE THREAD

**Issue by jlmelville** [rois-codh/kmnist#5]
There are some duplicate images between the training set and test set (as downloaded on 28 December). In case this isn't intentional (maybe someone had very reproducible hand writing on the scale of the resolution of MNIST?) the pairs are:

training image 21761 and test image 1951
training image 22145 and test image 3857
training image 34825 and test image 6844
training image 38534 and test image 9912
training image 39317 and test image 5343
training image 41654 and test image 6051

this is using the same (0-based) indexing as #1.

**Comment by tkasasagi** [comment]
Thank you for addressing the issue. I check it out.

**Comment by mxbi** [comment]
Updated dataset released back in February fixes this. Thank you again for reporting this!
